revolv/static/adjustment.scss

```diff
@@ -0,0 +1,25 @@
+@import "utils";
+@import "revolv-settings";
+
+.save-button {
+	@include revolv-hover-button($revolv-dark-blue, $revolv-dark-blue);
+	width: 97%;
+	margin-right: 15px;
+	margin-bottom: 30px;
+}
+
+.container {
+	margin-top: 50px;
+}
+
+body {
+	position: inline;
+}
+
+#id_amount {
+	position: absolute;
+	margin-top: -45px;
```

Comment:
Also, wat, this seems kind of wonky, is there a reason it needs to have negative margin? If it's position: absolute shouldn't you just define top and left?